modules/floating-buttons/assets/js/shared/frontend/handlers/click-tracking.js

```diff
@@ -66,7 +66,7 @@ export default class ClickTrackingHandler extends Base {
 
 		this.clicks.push( documentId );
 
-		if ( this.clicks.length >= 10 ) {
+		if ( this.clicks.length >= 2 ) {
```

Comment:
@louiswol94 this is for testing purposes, maybe should I put it under control of a querystring parameter?

Reply:
@louiswol94 I could click 10 times, but TBH 10 is an arbitrary number we choose, not sure what makes more sense here. `beforeunload` doesn't seem to fire even if I read that it should fire.

Reply:
@nicoladj77 As I see it, then we don't have to change anything here 🤔

Reply:
@louiswol94 rolled back changes